inst/include/dust/dust.hpp

```diff
@@ -324,6 +324,22 @@ class Dust {
     }
   }
 
+  std::vector<real_t> simulate(const std::vector<size_t>& step_end) {
```

Comment:
So this now using multi_pars, rather than `std::vector<dust::pars_t<T>>& pars`? Seems like a good idea

Reply:
What's the difference now between `simulate` and `run` on the dust object? Perhaps a comment could be added?